Core/OfficeDevPnP.Core/packages.config

```diff
@@ -7,6 +7,7 @@
   <package id="Microsoft.Data.Services.Client" version="5.6.2" targetFramework="net45" />
   <package id="Microsoft.Graph" version="1.1.1" targetFramework="net45" />
   <package id="Microsoft.Graph.Core" version="1.2.1" targetFramework="net45" />
+  <package id="Microsoft.Identity.Model.Extensions" version="2.0.1459.0" targetFramework="net45" />
```

Comment:
Please remove this nuget reference from the project as it is references a repacked Microsoft official assembly. The nuget package has been released by an individual not associated with Microsoft. If you remove the nuget package I'm happy to reevaluate the PR. Thanks :-)